test/ApplicationMarshalMiddlewarePipeTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive;
+
+use PHPUnit_Framework_TestCase as TestCase;
```

Comment:
2.0 release only at this point; I'd rather not duplicate the changes here, as it makes merging more difficult.